rpcs3/rpcs3qt/tooltips.h

```diff
@@ -58,6 +58,7 @@ class Tooltips : public QObject
 		const QString ppu_precise               = tr("Interprets PPU code with absolute accuracy.\nThis is the most accurate Interpreter, but very slow to play games with.\nYou may try this as a last resort if you encounter odd bugs or crashes.\nIf unsure, use PPU Interpreter Fast or PPU Recompiler (LLVM).");
 		const QString ppu_fast                  = tr("Interprets PPU code with sacrificed accuracy in order to achieve better performance.\nThis is the fastest interpreter.\nIt very rarely breaks games even in comparison to the Precise option.\nTry this if PPU Recompiler (LLVM) fails.");
 		const QString ppu_llvm                  = tr("Recompiles and caches the game's PPU code using the LLVM Recompiler once before running it for the first time.\nThis is by far the fastest option and should always be used.\nShould you face compatibility issues, fall back to one of the Interpreters and retry.\nIf unsure, use this option.");
+		const QString ppu_precompilation        = tr("Precompile PRX and OVL executables in PPU LLVM mode at startup. (before they are used)\nIf disabled these executables will compile only when used in-game (pausing the game unexpectedly and possibly frequently).\nDisabling is recommnded only if you want to quickly test a game without long playing of it.");
```

Comment:
Maybe this sounds better
```suggestion
		const QString ppu_precompilation        = tr("Searches the game's directory and precompiles extra PPU modules during boot.\nIf disabled, these executables will only compile when needed. Sepending on the game, this might interrupt the gameplay unexpectedly and possibly frequently.\nOnly disable this if you want to get ingame more quickly.");
```